anomaly-benchmarking/generate-datasets/script.whizzml

```diff
@@ -131,16 +153,16 @@
 ;; possible. Generates the max possible datasets (limit 40) at those
 ;; frequencies. Returns a list of datasets.
 
-(define (relative-frequency ds-id normal-rows)
+(define (relative-frequency ds-id normal-rows replicates freq-list)
   (let (ds (fetch ds-id)
         rows (ds "rows")
         ds-name (ds "name")
         my-tags (ds "tags")
         max-freq (/ rows (+ rows normal-rows)))
-    (for (freq [0.001 0.005 0.01 0.05 0.1])
+    (for (freq freq-list)
       (let (k (/ (* freq normal-rows) (- 1 freq)))
```

Comment:
Not sensibly. It is the relative frequency of anomalies, so if it is 1 that means that dataset is all anomalies and no normal points. Should I put in a catch in case someone is trying to use 1 without thinking?